deploy/ocm-agent-custom-resource/10-ocm-agent-cr.yaml

```diff
@@ -0,0 +1,13 @@
+apiVersion: ocmagent.managed.openshift.io/v1alpha1
```

Comment:
the directory to place those files, should be better to rename to `ocm-agent-operator-resources`? just like the other managed service. I am not pretty sure about this, maybe discuss within team

Reply:
+1 I prefer `ocm-agent-operator-resources`

Reply:
Thank you for review. Updated the directory name to `ocm-agent-operator-resources`